### ISSUE TITLE
cli version datatable not working

### ISSUE THREAD

**Issue by adaluo** [creativetimofficial/material-dashboard-pro-angular#20]
the datatable is not working in the new update in cli version.

**Comment by alexandru-paduraru** [comment]
@adaluo thank you for pointing out. We are doing some tests to see what is causing the issue, the datatables was working fine in the v1.1.2 update.

**Comment by Delysid** [comment]
I have the same issue in v.1.2.0. It's works good in dev mode. But in production it causes an error.

```
ERROR in Type MdTableComponent in ./src/app/md/md-table/md-table.component.ts is part of the declarations of 2 modules: LbdModule in ./src/app/md/md.module.ts and DashboardModule in ./src/app/dashboard/dashboard.module.ts! Please consider moving MdTableComponent in ./src/app/md/md-table/md-table.component.ts to a higher module that imports LbdModule in ./src/app/md/md.module.ts and DashboardModule in ./src/app/dashboard/dashboard.module.ts. You can also create a new NgModule that exports and includes MdTableComponent in ./src/app/md/md-table/md-table.component.ts then import that NgModule in LbdModule in ./src/app/md/md.module.ts and DashboardModule in ./src/app/dashboard/dashboard.module.ts.

ERROR in ./src/main.ts
Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in './src'
 @ ./src/main.ts 4:0-74
 @ multi ./src/main.ts
```

**Comment by alexandru-paduraru** [comment]
@adaluo and @Delysid we fixed the issue with the Datatables in v1.2.1, we switched to CLI and discontinued the ng2 version. CLI is way better and simple to be used. We also added in the documentation a tutorial on how to prepare the package.json to work smooth on the Production. Please let us know if you have any further feedback.
